hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -252,5 +256,25 @@ genCloseMutation (tx, _utxo) =
             let closingTime = slotNoToUTCTime healthySlotNo
              in posixFromUTCTime $ addUTCTime (fromInteger contestationPeriod) closingTime
         , headId = toPlutusCurrencySymbol Fixture.testPolicyId
+        , contesters
+        }
+    st -> error $ "unexpected state " <> show st
+
+  mutateClosedContesters :: Gen (TxOut CtxTx)
+  mutateClosedContesters = do
+    n <- elements [1, 5]
+    hashes <- vectorOf n genHash
```

Comment:
Could: use arbitrary sizes of key lists > 0 to make it clear what is accepted (only 0)